docs/how-to/load-models-from-pvc.md

```diff
@@ -0,0 +1,45 @@
+# Load Models from PVC
+
+You can store your models in a Persistent Volume Claim (PVC) and let KubeAI use them for serving.
+
+You must ensure the model files are already present in the PVC before creating the Model resource.
```

Comment:
Add a blurb and link to the alternative approach where KubeAI does this for you (cacheProfiles).

Reply:
done